core/shared/src/main/scala/fs2/text.scala

```diff
@@ -21,153 +21,174 @@
 
 package fs2
 
-import java.nio.{Buffer, CharBuffer}
+import java.nio.{Buffer, ByteBuffer, CharBuffer}
 import java.nio.charset.Charset
 
 import scala.collection.mutable.{ArrayBuffer, Builder}
 import scodec.bits.{Bases, ByteVector}
 
 /** Provides utilities for working with streams of text (e.g., encoding byte streams to strings). */
 object text {
-  private val utf8Charset = Charset.forName("UTF-8")
-  private[this] val utf8BomSeq: Seq[Byte] = Array(0xef.toByte, 0xbb.toByte, 0xbf.toByte).toSeq
 
-  /** Converts UTF-8 encoded byte stream to a stream of `String`. */
-  def utf8Decode[F[_]]: Pipe[F, Byte, String] =
-    _.chunks.through(utf8DecodeC)
+  object utf8 {
+    private val utf8Charset = Charset.forName("UTF-8")
```

Comment:
Doesn't really matter, but the JDK already provides this as a constant: `StandardCharsets.UTF_8`.

Reply:
Showing my age - I avoided that when we wrote this stuff initially (in scodec) to avoid the dependency on Java 7. :)